tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/configurables/ConfigurableTest.java

```diff
@@ -244,6 +245,22 @@ public void testInvalidDefaultableField() throws BallerinaTestException {
         errorLog.waitForText(5000);
     }
 
+    @Test
+    public void testSchedulerThreadPoolSize() throws BallerinaTestException {
```

Comment:
I think this is not part of configuration tests. Shall we move to a separate class.

Reply:
runtime.SchedulerConfigTest

Reply:
Should we need 2 tests. 
1 warning and take the env value
2 take from configurable